tendermint/src/lite/verifier.rs

```diff
@@ -566,8 +579,8 @@ mod tests {
     }
 
     // valid to skip, but invalid commit. 1 validator.
-    #[test]
-    fn test_verify_single_skip_1_val_verify() {
+    #[tokio::test]
+    async fn test_verify_single_skip_1_val_verify() {
```

Comment:
Do the tests which don't use async functions need to be async?

Reply:
No, they don't. Maybe this wasn't intentional (only the bisection test use async functions).

Reply:
https://github.com/informalsystems/tendermint-rs/pull/195